contrib/cirrus/runner.sh

```diff
@@ -96,8 +100,12 @@ function _run_bindings() {
 }
 
 function _run_docker-py() {
+    m="Testing of API was performed using the **DOCKER** client"
+    warn "$m"
     source venv/bin/activate
-    make run-docker-py-tests
+    if ! make run-docker-py-tests; then
```

Comment:
@edsantiago should this go through the logformatter as well?

Reply:
No idea. Test is running, I'll look at results once it finishes and let you know.

Reply:
Update: nope, doesn't look like something logformatter will handle right now. (Maybe the one I did last week uses `test -v`, and this one has no `-v`?) Anyway, this LGTM, thank you!

Reply:
Great, thanks.